activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -172,6 +177,21 @@ def table_comment(table_name) # :nodoc:
           end
         end
 
+        # Returns the partition definition of a given table
+        def table_partition_definition(table_name) # :nodoc:
+          scope = quoted_scope(table_name, type: "BASE TABLE")
+          return unless scope[:name]
+
+          query_value(<<~SQL.squish, "SCHEMA")
```

Comment:
`squish` will make this SQL query as a long single line, I think this is not needed here. Similar to other queries in this file.